baseplate/context/memcache/lib.py

```diff
@@ -0,0 +1,233 @@
+import logging
+import json
+import zlib
+
+from ..._compat import (
+    long,
+    pickle,
+    string_types,
+)
+
+
+"""Memcache serialization/deserialization (and compression) helper methods.
```

Comment:
if you want this module to get a page in the docs, you'll need to add a new or update an existing `.rst` in the `doc/` subtree.